Model/Resolver/setRzpPaymentDetailsForOrder.php

```diff
@@ -0,0 +1,210 @@
+
+    /**
+     * @param GetCartForUser $getCartForUser
+     * @param SetPaymentMethodOnCartModel $setPaymentMethodOnCart
+     * @param CheckCartCheckoutAllowance $checkCartCheckoutAllowance
+     * @param PaymentMethod $paymentMethod
+     * @param \Magento\Sales\Api\Data\OrderInterface $order
+     * @param \Razorpay\Magento\Model\Config $config
+     * @param \Magento\Sales\Model\Service\InvoiceService $invoiceService
+     * @param \Magento\Framework\DB\Transaction $transaction
+     * @param \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+     */
+    public function __construct(
+        GetCartForUser $getCartForUser,
+        SetPaymentMethodOnCartModel $setPaymentMethodOnCart,
+        CheckCartCheckoutAllowance $checkCartCheckoutAllowance,
+        PaymentMethod $paymentMethod,
+        \Magento\Sales\Api\Data\OrderInterface $order,
+        \Razorpay\Magento\Model\Config $config,
+        \Magento\Sales\Model\Service\InvoiceService $invoiceService,
+        \Magento\Framework\DB\Transaction $transaction,
+        \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+    ) {
+        $this->getCartForUser             = $getCartForUser;
+        $this->setPaymentMethodOnCart     = $setPaymentMethodOnCart;
+        $this->checkCartCheckoutAllowance = $checkCartCheckoutAllowance;
+        $this->rzp                        = $paymentMethod->rzp;
+        $this->order                      = $order;
+        $this->config                     = $config;
+        $this->invoiceService             = $invoiceService;
+        $this->transaction                = $transaction;
+        $this->scopeConfig                = $scopeConfig;
+        $this->_objectManager             = \Magento\Framework\App\ObjectManager::getInstance();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function resolve(Field $field, $context, ResolveInfo $info, array $value = null, array $args = null)
+    {
+        if (empty($args['input']['order_id'])) {
+            throw new GraphQlInputException(__('Required parameter "order_id" is missing.'));
+        }
+
+        $order_id = $args['input']['order_id'];
+        if (empty($args['input']['rzp_payment_id'])) {
+            throw new GraphQlInputException(__('Required parameter "rzp_payment_id" is missing.'));
+        }
+
+        $rzp_payment_id = $args['input']['rzp_payment_id'];
+
+        if (empty($args['input']['rzp_signature'])) {
+            throw new GraphQlInputException(__('Required parameter "rzp_signature" is missing.'));
+        }
+
+        $rzp_signature = $args['input']['rzp_signature'];
+
+        $rzp_order_id = '';
+        try {
+            $collection     = $this->_objectManager->get(\Magento\Sales\Model\Order::class)
+            ->getCollection()
+            ->addFieldToSelect('*')
+            ->addFilter('increment_id', $order_id)
+            ->getFirstItem();
+            $salesOrder = $collection->getData();
```

Comment:
@enishant , line gap. Please follow across the changes.

Reply:
@ramth05 Added line gap across the changes